portfolio/src/main/webapp/style.css

```
@@ -1,9 +1,35 @@
 #content {
+  color: white;  
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  text-align: justify;
+  width: 700px;
+
+}
+
+body{
```

Add a space after the selector (https://google.github.io/styleguide/htmlcssguide.html#Declaration_Block_Separation)
```suggestion
body {
```